assignment1.txt

```diff
@@ -0,0 +1,10 @@
+1.  A compiler turns the coding which is readable for humans into a binary form, so that a computer can read it.
```

Comment:
right, more specifically, a compiler takes your code written in C, removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.